airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -137,4 +312,40 @@ public TemporalClient(final WorkflowClient client, final Path workspaceRoot) {
     return new TemporalResponse<>(operationOutput, metadata);
   }
 
+  /**
+   * Check if a workflow is currently running. It is using the temporal pagination (see:
+   * https://temporalio.slack.com/archives/CTRCR8RBP/p1638926310308200)
+   */
+  public boolean isWorkflowRunning(final String workflowName) {
+    ByteString token;
+    ListOpenWorkflowExecutionsRequest openWorkflowExecutionsRequest =
+        ListOpenWorkflowExecutionsRequest.newBuilder()
+            .setNamespace(client.getOptions().getNamespace())
+            .build();
```

Comment:
```suggestion
      ListOpenWorkflowExecutionsRequest.newBuilder().setNamespace(client.getOptions().getNamespace()).build();
```

Reply:
Same than https://github.com/airbytehq/airbyte/pull/8352#discussion_r774495911 I really feel that the multi line is more readable especially with 2 imbrication levels.